slurm/grpo.slurm

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+#SBATCH --job-name=open-r1-grpo
+#SBATCH --nodes=1
+#SBATCH --ntasks-per-node=1
+#SBATCH --exclusive
+#SBATCH --gres=gpu:8
+#SBATCH --partition=hopper-prod 
+#SBATCH --output=./logs/%x-%j.out
+#SBATCH --err=./logs/%x-%j.err
+
+set -x -e
+
+source ~/.bashrc
+conda activate openr1
+echo "START TIME: $(date)"
+echo "PYTHON ENV: $(which python)"
+
+MODEL_PATH=$1
+DATASET_PATH=$2
+ACCELERATOR=$3
+
+# Training setup
+NUM_NODES=$SLURM_NNODES
+GPUS_PER_NODE=8
+WORLD_SIZE=$(($NUM_NODES*$GPUS_PER_NODE))
+
+
+# so processes know who to talk to
+MASTER_ADDR=$(scontrol show hostnames $SLURM_JOB_NODELIST | head -n 1)
+MASTER_PORT=6000
+
+export CMD=" \
+    src/open_r1/grpo.py \
+    --model_name_or_path $MODEL_PATH \
+    --dataset_name $DATASET_PATH \
+    --learning_rate 2.0e-5 \
+    --num_train_epochs 1 \
+    --max_completion_length 1024 \
+    --max_prompt_length 512 \
+    --per_device_train_batch_size 4 \
+    --per_device_eval_batch_size 4 \
+    --gradient_accumulation_steps 4 \
+    --gradient_checkpointing \
+    --bf16 \
+    --use_vllm \
+    --vllm_device auto \
+    --vllm_gpu_memory_utilization 0.7 \
+    --logging_steps 5 \
+    --eval_strategy steps \
+    --eval_steps 100 \
+    --output_dir data/Qwen2.5-1.5B-Open-R1-GRPO
+    "
+
+export LAUNCHER="HF_HUB_ENABLE_HF_TRANSFER=1 ACCELERATE_LOG_LEVEL=info TRANSFORMERS_VERBOSITY=info accelerate launch \
+    --config_file configs/$ACCELERATOR.yaml  \
```

Comment:
not sure if this works, but i think we need to tweak num processes to be N-1 devices
```suggestion
    --config_file configs/$ACCELERATOR.yaml  \
    --num_processes $WORLD_SIZE - 1 \
```